egs/thchs30/s5/local/dae/add-noise-mod.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import optparse
+import random
+import bisect
+import re
+import logging
+import wave
+import math
+import struct
+import sys
+import os
+
+try:
+  import pyximport; pyximport.install()
+  from util import *
```

Comment:
OK, now I understand -- I was wrongly assuming util is a python system package. Perhaps rename it to less generic, e.g. tsch30_util (or something like that)?